api/services/common/WebhookService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using LiteralLifeChurch.LiveStreamingApi.enums;
+using LiteralLifeChurch.LiveStreamingApi.exceptions;
+using LiteralLifeChurch.LiveStreamingApi.models.output;
+using Newtonsoft.Json;
+using RestSharp;
+using System;
+using System.Collections.Specialized;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace LiteralLifeChurch.LiveStreamingApi.services.common
+{
+    public class WebhookService : IService
+    {
+        public async Task CallWebhookAsync(Uri uri, ActionEnum action, ResourceStatusEnum status)
+        {
+            if (uri == null) return;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5510625)